apps/web/vite.config.ts

```diff
@@ -1,36 +1,41 @@
 import { tamaguiPlugin } from '@tamagui/vite-plugin'
 import react from '@vitejs/plugin-react-swc'
 import path from 'path'
-import { defineConfig } from 'vite'
+import { defineConfig, loadEnv } from 'vite'
 
 import type { PluginOption } from 'vite'
 
-process.env.TAMAGUI_TARGET = 'web'
-process.env.TAMAGUI_DISABLE_WARN_DYNAMIC_LOAD = '1'
-
 const tamaguiConfig = {
   components: ['@status-im/components'],
   config: './tamagui.config.ts',
   // useReactNativeWebLite: true,
 }
 
 // @see: https://vitejs.dev/config
-export default defineConfig({
-  resolve: {
-    // mainFields: ['module', 'jsnext:main', 'jsnext'],
-    alias: {
-      '@status-im/components/hooks': path.resolve(
-        '../../packages/components/hooks'
+export default defineConfig(({ mode }) => {
+  const env = loadEnv(mode, process.cwd(), '')
+
+  return {
+    resolve: {
+      // mainFields: ['module', 'jsnext:main', 'jsnext'],
+      alias: {
+        '@status-im/components/hooks': path.resolve(
+          '../../packages/components/hooks'
+        ),
+        '@status-im/components': path.resolve('../../packages/components/src'),
+      },
+    },
+    define: {
+      global: 'window',
+      'process.env.TAMAGUI_TARGET': JSON.stringify(env.TAMAGUI_TARGET),
+      'process.env.TAMAGUI_DISABLE_WARN_DYNAMIC_LOAD': JSON.stringify(
```

Comment:
where did you get that these vars are runtime from please?

Reply:
[@]prichodko

Reply:
I saw it in the Tamagui's [starters](https://github.com/tamagui/tamagui/tree/master/starters) folder. I don't see it there anymore, so let's drop and test it.